talentsearch/resources/js/components/search/EstimatedCandidates.tsx

```diff
@@ -41,13 +47,13 @@ const EstimatedCandidates: React.FunctionComponent<
           {intl.formatMessage(
             {
               defaultMessage:
-                "There are approximately <span>{totalEstimatedCandidates}</span> candidates right now who meet your criteria.",
+                "There are approximately <weight>{candidateCount}</weight> candidates right now who meet your criteria.",
```

Comment:
I like this solution but should it be _wait_?

Reply:
I put it as weight cause it was changing the font-weight.

Reply:
Oh, haha, I thought it was for the wait spinner.  😳 No worries then.

Reply:
I'd use <bold> instead, its a bit more descriptive